README.md

```diff
@@ -120,20 +114,17 @@ sudo yum install tezos-client-babylonnet
 
 ## Other Linux distros usage
 
-Build static `tezos-client` binary or download it from the CI.
+Build static `tezos-*` binary or download it from the CI.
```

Comment:
I think from now on we can assume that CI artifacts are only for internal usage by maintainers of this repo and people should download stuff from releases.

So here I would suggest downloading from releases rather than CI.

As a side note, I think the order of entities in README is quite wrong:
1. Usage instructions should be before build instructions.
2. For Ubuntu and Fedora we should suggest PPA/copr before downloading `.deb` or `.rpm` packages.

And btw, there we also suggest downloading from CI rather than releases.